bundles/com.salesforce.bazel.eclipse.core/src/com/salesforce/bazel/eclipse/core/model/discovery/JavaAspectsClasspathInfo.java

```diff
@@ -410,7 +410,13 @@ public CompileAndRuntimeClasspath compute() throws CoreException {
 
         // Collect jars referenced by runtime deps
         for (TargetKey targetKey : runtimeDeps) {
-            var entries = resolveDependency(targetKey);
+            Collection<ClasspathEntry> entries;
+            try {
+                entries = resolveDependency(targetKey);
+            } catch (Exception e) {
+                LOG.error("Error resolving runtime dependency: {}", targetKey, e);
+                continue;
+            }
```

Comment:
Please remove those. The exceptions needs to be surfaced so they end up in the Eclipse error log. `LOG.error` is only good in the LS case but not ideal for Eclipse UI (where the plugin is used as well).

Can you open a separate issue for the problem discovered here?
Skipping individual jars might make sense but the classpath would be broken and unusable anyway.